client/bower.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Sunfish",
+  "description": "Sunfish a file sharing app",
+  "version": "0.0.0",
+  "homepage": "https://github.com/NebulousLabs/Sunfish",
+  "dependencies": {
+    "angular": "~1.4.0",
+    "angular-route": "~1.4.0",
+    "foundation": "~5.5.2",
+    "jquery": "2.1.1"
+  }
+}
```

Comment:
As much as possible, we should try to keep our dependencies the same across projects. The UI uses pure.css instead of foundation.